exonum-java-binding/cryptocurrency-demo/pom.xml

```diff
@@ -194,7 +195,7 @@
           <archive>
             <manifestEntries>
               <Plugin-Id>${artifactName}</Plugin-Id>
```

Comment:
(here and elsewhere in services) PluginId is not correct (must be rid:name:version) — see the task desc.